tox.ini

```diff
@@ -184,3 +186,6 @@ extend-ignore = E501,E203,W503
 
 [pep8]
 ignore = E265,E501
+
+[testenv:.tox]
+envdir = {toxworkdir}/{envname}
```

Comment:
The top-level testev doesn't set envdir, so this is not required.

Reply:
reverted this change